page-changelog.php

```diff
@@ -3,6 +3,80 @@
 </div>
 <p>We recommend you follow our <a href="/release-channels">Release Channels</a> to always have the latest ownCloud Server version which fits your needs.</p>
 
+<h3>Version 8.2.1 <small>November 17 2015</small></h3>
+<ul>
+<li>Sharing improvements:</li>
+<ul>
+<li>Show path when sharing with owner</li>
+<li>Enable pagination on public link share page</li>
+<li>Added error numbers in the sharing error message</li>
+<li>Deal with NoUserException in sharing code</li>
+<li>Fix cannot change share info after switching sections</li>
+<li>Fix changing expiration date of shared link breaks password</li>
+<li>Various other fixes</li>
+</ul>
+<li>Performance improvements:</li>
+<ul>
+<li>Optimize multiple shared locks for a single process</li>
+<li>Don't lock if we're only reading cache metadata</li>
+<li>Escape like parameter in cache operations</li>
+</ul>
+<li>Storage improvements:</li>
+<ul>
+<li>Improve dealing with Dropbox and Google Drive</li>
+<li>Moving files makes them disappear (SWIFT object store)</li>
+<li>Transactional file locking database backend warning is only shown when there is another warning.</li>
+<li>Don't lock /$user/files</li>
+<li>Include the final update in the transaction when moving a folder in the cache</li>
+<li>WebDAV MOVE on a non existing file results in Internal Server Error #20069</li>
+<li>Catch all exception if table doesn't exist #19884 #19893</li>
+<li>Various other objectstore fixes</li>
+</ul>
+<li>Usability and UI:</li>
+<ul>
+<li>Made error message about file not found more specific in federated sharing</li>
+<li>Improved CSS, fonts, text color on various buttons, login screen, menu and settings</li>
+<li>Hide strength indication after password change and hide notifications after time-out or on delete</li>
+<li>Fixes to inconsistent language and translation support, show language code for unknown languages</li>
+<li>Fix icons for share/public folders, fix spinner positions in share tab</li>
+<li>Close user menu when clicking on other menus</li>
+<li>Sidebar fixes and hiding sidebar for trashbin view</li>
+<li>Move alt text for favorite action to image</li>
+<li>Sidebar should not open, when renaming a file on mobile</li>
+<li>Do not register sidebar panels when no sidebar</li>
+<li>Update process: state which step we are going to start and warn if it might be slow</li>
+</ul>
+<li>User Management and LDAP fixes:</li>
+<ul>
+<li>LDAP fixes for quota, user mapper, initial user creation and connectivity</li>
+<li>Fix group admin settings, group assignment when group name is a number</li>
+<li>Enable proper CardDAV cookie authentication</li>
+</ul>
+<li>Documentation, reliability and stability fixes</li>
```

Comment:
All the other sub-heading text ends in a colon, except this one here. The text parser in my brain had a stack dump when it got to here :)